### DISCUSSION TITLE
code字体建议改成Jetbrains Mono增加阅读体验

### DISCUSSION THREAD

**Discussion by 12og3r** [compose-museum/jetpack-compose-tutorial#36] [Ideas]
代码背景色可以考虑改成合适的暗色，更加醒些🥵

**Comment by totoneei** [comment]
背景色参考的安卓官方文档

**Comment by totoneei** [comment]
背景涩你看着调一下也可以

**Comment by 12og3r** [comment]
好捏，整张涩图捏

**Comment by totoneei** [comment]
不过我感觉似乎粗体还没有设置成功，以及代码大小可以改大一些，但是我不会力，我太菜了

**Comment by totoneei** [comment]
@RugerMcCarthy

**Comment by 12og3r** [comment]
CSS选择器对code标签设置字体样式不可以咩

**Comment by totoneei** [comment]
https://squidfunk.github.io/mkdocs-material/setup/changing-the-fonts/

**Comment by totoneei** [comment]
俺试过好像 8 太行

**Comment by 12og3r** [comment]
![image](https://user-images.githubusercontent.com/30253468/118235719-b9956780-b4c7-11eb-8ae1-fcb16333633f.png)
这样可以咩

**Reply by 12og3r** [reply]
你要对所有字体粗细进行修改，还是仅对代码

**Reply by totoneei** [reply]
代码字体就可以了
